pkgs/development/node-packages/node-packages.json

```diff
@@ -17,6 +17,7 @@
 , "bower2nix"
 , "browserify"
 , "castnow"
+, {"cdktf-cli": "next"}
```

Comment:
Just per [upstream-docs](https://learn.hashicorp.com/tutorials/terraform/cdktf-install):

> Because cdktf is being constantly improved, we recommend keeping up to date with the cutting edge development version by using the @next tag.

But all the markdown docs living in their repo don't point to `next`, so I can change that to point to stable.

Reply:
I believe it's better to stick with proper releases in this case, considering that [cdktf publishes releases on a frequent basis][1]. For packages that do use unstable versions, it's typically the case that the upstream doesn't have a release or have ceased to publish one. It's also going to be harder to maintain unstable releases for packages in `nodePackages` because they all have to be upgraded at once.

[1]: https://github.com/hashicorp/terraform-cdk/releases

Reply:
Makes sense to me, and thanks much for the feedback. I've amended the commit and rebased from current HEAD of the master branch.